registry/server/errorHandler/preProcessErrorResponse.ts

```diff
@@ -0,0 +1,10 @@
+const preProcessErrorResponse = (error: Error, errorInfo = {}) => {
```

Comment:
I don't see a big value in this function... IMO it causes more issues then it solves

Reply:
Removed